UI/src/components/features/RipaHomeContainer.vue

```diff
@@ -103,7 +106,17 @@ export default {
       'officerId',
       'agency',
       'mappedGpsLocationAddress',
+      'isAuthenticated',
     ]),
+    getAuthAndLocalStorageCheck() {
+      // if the user is NOT authenticated AND does not have a local storage cache
+      // that means they haven't logged in and must reauthenticate
+      if (!this.isAuthenticated && !localStorage.getItem('ripa_cache_date')) {
```

Comment:
We probably should use the existing isOnlineAndAuthenticated since we don't want to check if the user is offline.

Reply:
I use this check to see if you should show the buttons to get into the form.  I thought if you were offline you could still start a form.  So if isOnline is false, the user should still see the three buttons which let them log a stop.

Reply:
That makes sense Todd